contracts/triggers/LiquityRatioTrigger.sol

```diff
@@ -30,6 +33,8 @@ contract LiquityRatioTrigger is ITrigger, AdminAuth, LiquityRatioHelper {
         (uint256 currRatio, bool isActive) = getRatio(triggerSubData.troveOwner);
         
         if (isActive == false) return false;
```

Comment:
Would add `if (isActive == false || currRatio == 0)` just as a sanity check, shouldn't really happen but extra validation is not costly in this situation